src/dbnode/storage/series/buffer.go

```diff
@@ -614,7 +614,7 @@ func (b *dbBufferBucket) write(
 				// in a time window will still cause a flood of disk/CPU resource
 				// usage writing values to the commit log, even if the memory
 				// profile is lean as a side effect of this write being a no-op.
-				return nil
+				return false, nil
```

Comment:
Wanna delete the TODO right above this line since you're fixing it? :D

Reply:
Sounds good!